### DISCUSSION TITLE
Pixel tracking: a simple solution

### DISCUSSION THREAD

**Discussion by serrq** [tutao/tutanota#6017] [Ideas]
I propose automatically download all images files as soon as the email is delivered. Not when the user open it.

This provide a simple protection over real email opening, because nobody knowing exactly when a user open an email (it is an unintentional and automated action).

Of course download images doesn't mean open them for a security reason.

What do you think about it?

**Comment by charlag** [comment]
We discussed this few years before but as an option only. This should only be optional as it gives us access to a lot of user data. It will also incur significant costs for us.

**Comment by serrq** [comment]
Yes optionally and with some restrictions like data cap (up to 20mb) and for selected senders.
